sleap_io/model/video.py

```diff
@@ -0,0 +1,33 @@
+"""Data model for Videos.
+
+Videos are SLEAP data structures that store information regarding a video and its
```

Comment:
Videos -> The `Video` class (with backticks and then correct grammar plural to singular for rest of sentence)